ecommerce_extensions/tenant/extra_options.py

```diff
@@ -0,0 +1,60 @@
+"""
+This function gets called during every request by the
+extra_html and extra_scripts to validate all the custom html and scripts for a specific path.
+"""
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def validate_option_attributes(values, common_attributes, error_message):
```

Comment:
I have the same question that with `check_attribute`, why pass the error message as a param?
Since you are using the same error for both functions maybe you could make the error message a constant and declare it at the beginning of the file. Then you can use that constant inside both functions.